test_worker_node_restart_during_pvc_clone failing

Test case test_worker_node_restart_during_pvc_clone is failing with the error given below.


```
        if storage_type == "block":
            return (
>               self.pod_data.get("spec")
                .get("containers")[0]
                .get("volumeDevices")[0]
                .get("devicePath")
            )
E           TypeError: 'NoneType' object is not subscriptable

ocs_ci/ocs/resources/pod.py:279: TypeError
```


Test report https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-deploy-ocs-cluster/15364/testReport/

--- a/tests/manage/pv_services/pvc_clone/test_node_restart_during_pvc_clone.py
+++ b/tests/manage/pv_services/pvc_clone/test_node_restart_during_pvc_clone.py
@@ -168,9 +168,9 @@ class TestNodeRestartDuringPvcClone(ManageTest):
         # Verify md5sum
         for pod_obj in clone_pod_objs:
             file_name_pod = (
-                file_name
-                if (pod_obj.pvc.volume_mode == constants.VOLUME_MODE_FILESYSTEM)
-                else pod_obj.get_storage_path(storage_type="block")
+                pod_obj.get_storage_path(storage_type="block")
+                if (pod_obj.pvc.volume_mode == constants.VOLUME_MODE_BLOCK)
+                else file_name
             )
             pod.verify_data_integrity(
                 pod_obj,

